distribution/changelog.txt

```diff
@@ -23,6 +23,7 @@
 - Fix: [#21198] [Plugin] Setting brake or booster speeds on a tile element doesn’t work.
 - Fix: [#21290] Sound keeps playing when paused from fast-forward mode.
 - Fix: [#21291] Hungry guests heading to any flat ride do not count for warning threshold (original bug).
+- Fix: [#20631] IME window not positioned correctly.
```

Comment:
As I mentioned already, the changelog should be sorted by type first, then by ID. 20631 is lower than 21291.

Reply:
Sorry about that... Got there eventually!